nix/crates.nix

```diff
@@ -135,7 +135,7 @@ in {
         # Don't run the `jstz_api` integration tests until they've been paralellized
         #
         # Note: --workspace is required for --exclude. Once --exclude is removed, remove --workspace
-        cargoNextestExtraArgs = "--workspace --test \"*\" --exclude \"jstz_api\"";
+        cargoNextestExtraArgs = "--workspace --test \"*\" --exclude \"jstz_api\" --config-file ${src}/.config/nextest.toml";
```

Comment:
Won't this be needed for the `cargo-llvm-cov` check as well?

Reply:
Looking at `cargo llvm-cov test` the option isn't available, but it is for `cargo llvm-cov nextest`. #658 changes the command to `nextest` -- though this could be split out/included in this PR to avoid PR dependencies

Reply:
I see. Do you want to split that `nextest` change into an individual PR? You can apply `config-file` in that PR after this one is merged or I can wait for that and patch `cargo-llvm-cov` in this PR.